__Tests__/e2e.test.js

```diff
@@ -0,0 +1,19 @@
+const puppeteer = require('puppeteer');
+const URL = "http://127.0.0.1:5501/source/index.html";
+
+describe('basic user flow of the website', () => {
+    // Checking for 6 category cards on the website
+    it('Initial rendering of the page -- check the category cards', async () => {
+        const browser = await puppeteer.launch();
+        const page = await browser.newPage();
+        await page.goto(URL);
+      
+        // Get the all the category cards on the website
+        const numCategoryCards = await page.$$eval('category-card', (categoryCards) => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group23/cse110-fa21-group23/pullRequest?prid=8422468)
```suggestion
        const numCategoryCards = await page.$$eval("category-card", (categoryCards) => {
```